tensorly/regression/cp_plsr.py

```diff
@@ -34,7 +34,7 @@ class CP_PLSR:
     """
 
     def __init__(
-        self, n_components, tol=1.0e-9, n_iter_max=100, random_state=None, verbose=False
+        self, n_components, tol=1.0e-8, n_iter_max=100, random_state=None, verbose=False
```

Comment:
The interaction within a component was going on up to `n_iter_max` with the old tolerance and the tensorflow backend. I'm not sure why the accuracy is different in a backend-specific way, but this was helping with testing taking a more reasonable amount of time.